src/compile/header/index.ts

```diff
@@ -219,12 +193,17 @@ function getHeaderGroup(model: Model, channel: HeaderChannel, headerType: Header
   return null;
 }
 
-function getHeaderLabelProperties(config: Config) {
+export function getHeaderProperties(config: Config, facetFieldDef: FacetFieldDef<string>, properties: string[], propertiesMap: {[k in keyof HeaderConfig]: keyof VgTitleConfig}) {
```

Comment:
Should each property in `properties` be a key of HeaderConfig?

Reply:
The types only seem to work when `properties` is a `string[]` because the lists I pass in are created by calling `Object.keys` on a map, which seems to return a `string[]`. Is there a way around this?

Reply:
I changed the type using `util.keys` and it worked locally and passed header tests, but this change caused many of the other tests to fail because of type errors so I left it as is.